packages/cli/src/commands/website/setup.js

```diff
@@ -27,7 +28,12 @@ export default class WebsiteSetupCommand extends Command {
       const hasKey = keyValue != null && keyValue.length !== 0
       const keyPair = createKeyPair(hasKey ? keyValue : undefined, 'hex')
 
-      const address = pubKeyToAddress(keyPair.getPublic())
+      const address = pubKeyToAddress(
+        keyPair
+          .getPublic()
+          .encode()
+          .slice(1),
```

Comment:
Any reason to not handle the default encoding and require slicing?

Reply:
The idea was to keep these modules independent, here `keyPair.getPublic().encode().slice(1)` is specific to the fact we're using the `elliptic` package in `@erebos/secp256k1`, but `@erebos/keccak256` doesn't know about it.
Basically I wanted these packages to be as low-level as possible, with minimum dependencies.
In a way I also wanted to make using `@erebos/secp256k1` "non-default" so that developers really evaluate its usage and the security implication rather than being a "no-brainer" option.

Reply:
Ah, ok. Out of curiosity what are you slicing?

Reply:
It seems that slicing that byte is not specific to `elliptic` but is in fact specific to how ethereum addresses are determined. See [here](https://github.com/ethereum/go-ethereum/blob/391d4cb9b53eb9f71862cbb8a6cca332d5a76f94/crypto/crypto.go#L205):
```
Keccak256(pubBytes[1:])[12:]
```

Reply:
Yeah so the hex string of the pub key would always be prefixed `'0x04.....'` from my understanding